src/components/Jotting.svelte

```diff
@@ -74,14 +74,20 @@
 						<a href={getRelativeLocaleUrl(locale, `/jotting/${jotting.id.split("/").slice(1).join("/")}`)} class="c-primary font-600 link">{jotting.data.title}</a>
 					</span>
 					<span class="flex gap-1">
-						{#each jotting.data.tags as tag}
+						{#each jotting.data.tags || [] as tag}
```

Comment:
Same change needed as in `Note.svelte`.
```suggestion
						{#each jotting.data.tags ?? [] as tag}
```